requirements.txt

```diff
@@ -0,0 +1,7 @@
+aiohttp~=3.8.1
+click~=8.0.3
+PyYAML~=6.0
+rich~=10.15.1
+setuptools~=57.4.0
+toml~=0.10.2
+yarl~=1.7.2
```

Comment:
why not https://github.com/neuro-inc/neuro-extras/blob/master/requirements/base.txt ?